src/Lean/Compiler/LCNF/Passes.lean

```diff
@@ -0,0 +1,20 @@
+/-
+Copyright (c) 2022 Henrik Böving. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Henrik Böving
+-/
+import Lean.Compiler.LCNF.PassManager
+import Lean.Compiler.LCNF.PullLetDecls
+import Lean.Compiler.LCNF.CSE
+
+namespace Lean.Compiler.LCNF
+
+@[cpass]
```

Comment:
I was expecting this line to compile only after the `update-stage0`.
How did you manage to compile this commit before the `update-stage0`?

Reply:
I didn't I just already had some parts of the code staged so i unstaged the stage0 stuff and did the full commit of the code first, I can flip them if you want.